package.json

```diff
@@ -64,7 +65,7 @@
         "eslint-plugin-prettier": "4.0.0",
         "husky": "7.0.4",
         "jsdoc-to-markdown": "7.1.0",
-        "lint-staged": "12.1.5",
+        "lint-staged": "12.1.7",
```

Comment:
any reason this was updated? dont see a note in the commits for it. if we don't warn about package changes, developers might not run `npm install` again, leading to issues.